src/Tools/dotnet-watch/src/Program.cs

```diff
@@ -38,6 +39,11 @@ public static async Task<int> Main(string[] args)
                 DebugHelper.HandleDebugSwitch(ref args);
                 using (var program = new Program(PhysicalConsole.Singleton, Directory.GetCurrentDirectory()))
                 {
+                    //checks if there is "run" in the args if NOT append a run as the last argument
+                    if (!args.Contains("run") && !args.Contains("test"))
```

Comment:
@eugeneogongo I would make it so that if no args are provided, it assumes run by default. Something like:

```suggestion
                    if (args.Length == 0)
                    {
                        args = new[] { "run" };
                    }
```

Reply:
Hello, @pranavkm , Thanks for the review. This seems fine when the user enters 
>dotnet watch

However, what about when the user specifies a path without "run" argument like below:
>dotnet watch --project  ./project/project.csproj

The above command has one argument. I think the command above **should run** without specifying the "run" argument. 
What your thoughts @pranavkm ?

Reply:
That's an interesting scenario. I feel that if the user was already specifying additional arguments, it seems like a small ask for them to consistently specify everything. The scenario we were targeting was to simplify the behavior for the most trivial case which is when you run `dotnet watch`

/cc @jkotalik since this was your suggestion, what do you think? Also @DamianEdwards

Reply:
My assumption was that anything `dotnet watch run` can do, `dotnet watch` does. So I'd expect `dotnet watch --project ./project/project.csproj` just to work and infer `dotnet watch run`.

Reply:
Fair enough. Could we change the check to verify that there are no args, or the only orgs that are present are `--` ones?

Reply:
@pranavkm , 
>Could we change the check to verify that there are no args, or the only orgs that are present start with-- prefix.

This will not be necessary since the Parser will handle no args  and args with a prefix situations. 
Simply, put we are translating dotnet watch to dotnet watch run.

Reply:
Checking for no command will be appropriate. The suggested code looks good..
Thank you. 🙏